src/Admin/Repository/SettingRepository.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * This file is part of OpenEMR.
+ *
+ * @package     OpenEMR
+ * @subpackage
+ * @link        https://www.open-emr.org
+ * @author      Robert Down <robertdown@live.com>
+ * @copyright   Copyright (c) 2019 Robert Down <robertdown@live.com
+ * @license     https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Admin\Repository;
+
+use Doctrine\ORM\EntityRepository;
+
+class SettingRepository extends EntityRepository
+{
+    public function getSettingByName($settingName) {
+        $results = $this->_em->getRepository($this->_entityName)->findBy(['name' => $settingName]);
```

Comment:
Out of curiousity, how does doctrine prevent sql injection?

Reply:
Doctrine uses parameter binding and escapes all inputs.  Pretty much every Object Relational Mapper (ORM) does that.